### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by baxmittens**
I just ran Grammarly on this paper.
Be very careful with accepting the changes.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ bibliography: paper.bib
 
 # Summary
 
-Spherical Harmonic Transforms (SHTs) can be seen as the spherical, two-dimensional, counterpart of Fourier Transforms, casting real-space data to the spectral domain and vice versa.
-As in Fourier analysis a function is decomposed into a set of amplitude coefficients, through an SHT, any spherically-symmetric field defined in real space can be decomposed into a set of complex harmonic coefficients $a_{\ell, m}$, commonly referred to as alms, each quantifying the contribution of the corresponding spherical harmonic function.
+Spherical Harmonic Transforms (SHT) can be seen as Fourier Transforms' spherical, two-dimensional counterparts, casting real-space data to the spectral domain and vice versa.
+As in Fourier analysis, a function is decomposed into a set of amplitude coefficients through an SHT, and any spherically-symmetric field defined in real space can be decomposed into a set of complex harmonic coefficients $a_{\ell, m}$, commonly referred to as alms, each quantifying the contribution of the corresponding spherical harmonic function.
```
Comment (LeeoBianchi) [inline paper/paper.md:28]: I think this rephrasing slightly changes the meaning of the sentence. I would keep the original.

**Review by LeeoBianchi — CHANGES_REQUESTED** [review]
Hi @baxmittens,

I went through the changed document and I would like to accept every change you are proposing except for two lines, where I left a comment. I would keep the original version in those two cases.

File: paper/paper.md (hunk 1 above)
Comment (baxmittens) [reply paper/paper.md:28]: just be careful with the plural. I think "counterpart`s`" is grammatically correct here, isn't it?

Comment (LeeoBianchi) [reply paper/paper.md:28]: > just be careful with the plural. I think "counterpart`s`" is grammatically correct here, isn't it?

Yes you're most probably right. Then I think we can accept the changes for line 27 while keeping line 28 as it is. What do you think?

**Comment by baxmittens** [comment]
I think you have to merge this pull request, right?

**Comment by LeeoBianchi** [comment]
> I think you have to merge this pull request, right?

I think that if I merge this one as it is I automatically accept **all** the changes.
If we want to proceed as we discussed, i.e. keeping those few lines unchanged you have to change the file and push a new commit on the branch to be merged. Am I wrong?

**Review by baxmittens — COMMENTED** [review]
ok

**Review by LeeoBianchi — APPROVED** [review]
Looks good now, thank you.

**Pull request merged**